.gitignore

```diff
@@ -4,12 +4,4 @@
 .direnv
 .vscode
 .idea
-test/test_kyber512
-test/test_kyber768
-test/test_kyber1024
-test/gen_KAT512
-test/gen_KAT768
-test/gen_KAT1024
-test/gen_NISTKAT512
-test/gen_NISTKAT768
-test/gen_NISTKAT1024
+bin/
```

Comment:
Any reason in particular why this was renamed? I think `test` is more suitable than `bin`.

Reply:
Mainly because I personally don't like having those binaries scattered in a folder alongside some C files. I prefer to keep those binaries under a separate directory. 

But I don't have a strong prefer over the folder name, or the folder location. I think I just followed some folder structure as `pqm4`.
And if we prefer to keep things under `test`, then I would prefer to keep them under `test/bin`.

Reply:
`test/bin` sounds good to me.